### ISSUE TITLE
Install Recommended plugins during import process

### ISSUE THREAD

**Issue by wpexplorer** [proteusthemes/one-click-demo-import#146]
It wouldn't be hard to add a function that runs before the import to activate any recommended plugins defined in the import_files array (different demos may recommend different plugins like WooCommerce, Events Calendar, etc). Installing plugins is as easy as visiting  admin_url( 'update.php' ) with the "install-plugin" action or running the "activate" action if the plugin is already installed.

This could prevent a lot of headaches for the end customer and make it easier to provide unique demos that make use of different plugins without the customer having to activate all possible plugins that may be used in any demo.

Are there any plans to integrate this? I am a bit strapped for time, but I could "sponsor" the update if there is a developer with some spare time available.

Thanks!

**Comment by wpexplorer** [comment]
Do you guys intend to stop updating/maintaining this plugin in exchange for MerlinWP. I am looking to invest time and money to support a quality demo importer for use in all future projects. I don't want to start sending out pull-requests for a project that is intended to be obsolete soon.

If you plan on keeping this project alive I don't mind submitting a pull request for the plugin installation functionality.

**Comment by capuderg** [comment]
No, we will not stop updating/maintaining this plugin. We still use this plugin in all our themes.

As for a PR for this issue... You are the first one to suggest this feature. All other theme authors (including us) are using TGMPA for plugin installation and we also warn the user to install all plugins before import. So I'm not sure this functionality is really needed + installing plugins without user knowledge or choice is not really a good idea. ( I would not agree to adding a control for plugin installation to our plugin, since we want this plugin to be focused only on one thing: import demo data)
